### PR TITLE
Gio/new storage

### PR DESCRIPTION
**Pull request by digmel**
local storage and dev page setup complete, added storybook, color, size, and button component boilerplate

### CODE REVIEW

File: src/components/button/Button.style.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    margin: Size.m,
+    justifyContent: 'center',
+  },
+
+  button: {
+    height: Size.xxl,
```
Comment (lasharela) [inline src/components/button/Button.style.ts:12]: `size` and `color` objects needs yo be with lower case letter

File: src/components/button/Button.style.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    borderRadius: Size.xxl,
+    backgroundColor: Color.primary,
+    justifyContent: 'center',
+    paddingLeft: 24.56,
+    paddingRight: 22.33,
+  },
+
+  title: {
+    color: Color.light,
+    fontWeight: '700',
```
Comment (lasharela) [inline src/components/button/Button.style.ts:23]: We don't have to use font weight.

We have fontFamily which is equivalent to 700

File: src/components/button/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import {Text, TouchableOpacity} from 'react-native';
+import {styles} from './Button.style';
+import {TButtonProps} from './Button.type';
+
+export const Button: FC<TButtonProps> = ({title, action}) => {
+  return (
+    <TouchableOpacity
+      style={styles.button}
+      onPress={() => {
+        action();
```
Comment (lasharela) [inline src/components/button/Button.tsx:11]: Action needs to be set directly 
`onPress={onPress}`

But even better - touchable opacity must have `{...props}` which includes onPress as well

File: src/components/config/Color.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+export const Color = {
```
Comment (lasharela) [inline src/components/config/Color.ts:1]: With lower.
Pnly components are with Capital letter. Rest is camelCase

File: src/components/config/Size.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+export const Size = {
```
Comment (lasharela) [inline src/components/config/Size.ts:1]: Same

File: src/components/index.ts [hunk 6]
```diff
@@ -12,6 +12,7 @@ export {default as InfoCard} from './InfoCard';
 export {default as CustomDrawerContent} from './CustomDrawerContent';
```
Comment (lasharela) [inline src/components/index.ts:12]: Let's update those as well

File: src/screens/development/shared/KeyValueList.style.ts [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+  container: {
+    marginBottom: 6,
+  },
+  rowContainer: {
+    flexDirection: 'row',
+
+    backgroundColor: 'gray',
```
Comment (lasharela) [inline src/screens/development/shared/KeyValueList.style.ts:10]: Replace gray with our `grayLight`

**Review by lasharela — CHANGES_REQUESTED** [review]

**Comment by digmel** [comment]
updated according to comments, button component will improved in next branch

**Review by lasharela — APPROVED** [review]

**Pull request merged**